spec/models/comment_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  user = User.create(name: 'Grabrielle', photo: 'mybaby.png', bio: 'A beautiful baby', post_counter: 0)
+  post = Post.create(user_id: author.id, title: 'Hello', text: 'This is my first post', likes_counter: 0,
+                     comments_counter: 0)
```

Comment:
Some tests for comments and likes are failing due to the following reasons.

![image](https://user-images.githubusercontent.com/38088818/228890115-bcf3c056-1fce-4747-a92d-273439d0e3d2.png)

-  On both the `comment_spec.rb` and `like_spec.rb` `line: 5` the variable `author` does not exit therefore `user_id: author.id` should be `user_id: user.id`.

Kindly fix all the issues with foreign keys on your test to make sure that all your tests pass.

Reply:
This issue is not yet resolved